secator/ai/actions.py

```
@@ -64,12 +67,95 @@ def get_query_engine(self):
+		hooks = _build_hooks_from_context(context)
+	except Exception as e:  # narrow to the rebuild — surface, don't degrade to hooks={}
+		if parent_has_drivers:
+			return {}, Warning(
+				message=f"Subagent spawn denied: persistence hook rebuild failed — {type(e).__name__}: {e}",
+				_context=context,
+			)
+		return {}, None
+	if parent_has_drivers and not hooks:
+		return {}, Warning(
+			message="Subagent spawn denied: parent has persistence drivers but child hook rebuild "
+					"was empty (would silently drop findings/docs)",
```

Flake8 (7.3.0)

[error] continuation line unaligned for hanging indent

(E131)